src/qtiCommonRenderer/renderers/interactions/ChoiceInteraction.js

```diff
@@ -186,18 +186,23 @@ const _setInstructions = function _setInstructions(interaction) {
 
     const highlightInvalidInput = function highlightInvalidInput($choice) {
         const $input = $choice.find('.real-label > input');
-        const $li = $choice.css('color', '#BA122B');
-        const $icon = $choice.find('.real-label > span').css('color', '#BA122B').addClass('cross error');
+        const $icon = $choice.find('.real-label > span');
+
+        const choiceStyle = $choice.attr('style');
+        const iconStyle = $icon.attr('style');
+        $choice.css('color', '#BA122B');
+        $icon.css('color', '#BA122B').addClass('cross error')
+
         let timeout = interaction.data('__instructionTimeout');
 
         if (timeout) {
             clearTimeout(timeout);
         }
         timeout = setTimeout(function () {
             $input.prop('checked', false);
-            $li.removeAttr('style');
-            $icon.removeAttr('style').removeClass('cross');
-            $li.toggleClass('user-selected', false);
+            $choice.attr('style', choiceStyle);
+            $icon.attr('style', iconStyle).removeClass('cross');
```

Comment:
**issue:** If `choiceStyle` and/or `iconStyle` are `undefined`, this will lead to `.attr()` being called as a getter and not as a setter. Therefore, the style won't be reset, and `.removeClass()` will fail. Actually, `.attr()` may return undefined if the attribute is not set, see https://api.jquery.com/attr/

**suggestion:** Make sure the variables `choiceStyle` and `iconStyle` have a value other than `undefined`, and/or use a different approach for resetting the style.

Reply:
I tried running $element.attr('style', undefined) before submitting the PR. It worked just fine.